lib/easypost/beta/referral_customer.rb

```diff
@@ -56,17 +56,17 @@ def create_easypost_credit_card(referral_api_key, stripe_object_id, priority = '
   end
 
   # Create a referral customer. This function requires the Partner User's API key.
-  # <b>DEPRECATED:</b> Please use <tt>Referral</tt> in the main namespace instead.
+  # <b>DEPRECATED:</b> Please use <tt>ReferralCustomer</tt> in the main namespace instead.
   def self.create(params = {}, api_key = nil)
-    warn '[DEPRECATION] Please use `Referral.create` in the main namespace instead.'
+    warn '[DEPRECATION] Please use `ReferralCustomer.create` in the main namespace instead.'
```

Comment:
question: do you plan to remove these separately? These deprecated referral customer functions can go since they are GA now and already exist in that namespace

Reply:
This file will be removed in the last PR after all the classes have been migrated to thread-safe